r-package/grf/R/input_utilities.R

```diff
@@ -276,3 +276,8 @@ observation_weights <- function(forest) {
   }
   observation.weight
 }
+
+do.call.grf = function(what, args, quote = FALSE, envir = parent.frame()) {
```

Comment:
For future generations, could you put a short comment here on why this function is necessary?